.github/workflows/workflow.yml

```diff
@@ -240,5 +324,23 @@ jobs:
         python --version
         pip install -e ".[dev]"  # install aspire
         pip freeze
+    - name: Restore Cache
```

Comment:
These steps are a bit repetitive. I guess there's no way to factor them out somehow into a custom action?

Reply:
Created issue #1229 to circle back on this.

Reply:
:+1: